### ISSUE TITLE
PET: Invalid pattern for ScanDate in PET schema

### ISSUE THREAD

**Issue by mnoergaard** [bids-standard/legacy-validator#1282]
Currently the ScanDate field is specified as the following in the PET schema:

```
"ScanDate": {
      "type": "string",
      "pattern": "date"
    },
```

However, this only allows for "date" to be entered as a valid ScanDate when validating.

A quick fix would be to update the code to the following:

```
"ScanDate": {
      "type": "string",
      "pattern": "^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(.\d+)?Z?$"
    },

```

Similar to in checkAcqTimeFormat.js. 

https://github.com/bids-standard/bids-validator/blob/1e92dc1da31046e741c80081cfb67e54ac11a763/bids-validator/validators/tsv/checkAcqTimeFormat.js#L5

**Comment by sappelhoff** [comment]
closed in #1283

**Comment by sappelhoff** [comment]
reopened because of https://github.com/bids-standard/bids-validator/pull/1283#issuecomment-844889275

**Comment by sappelhoff** [comment]
closed in #1290
